packages/types/src/atoms/graphql-types.ts

```diff
@@ -1177,9 +1205,12 @@ export type EhrAuditEventsArgs = {
   auditProfile?: Maybe<Scalars['String']>;
   count?: Maybe<Scalars['Int']>;
   cursorToken?: Maybe<Scalars['String']>;
+  from?: Maybe<Scalars['DateTimeOffset']>;
   pathwayId: Scalars['String'];
   pathwayType: Scalars['String'];
   patientGuid: Scalars['String'];
+  tags?: Maybe<Array<Maybe<Scalars['String']>>>;
```

Comment:
Side note to explain, I added these to our GraphQL queries back in ppm+ world but didnt think to regen types here. They are OK to bring in.